### PR TITLE
WT-11012 Add missing RTS tag in a log

### PR DESCRIPTION
**Pull request by etienneptl**
This adds:
- A new RTS tag for deleted pages.
- The prefix `ref=` in the log message to keep the RTS verifier happy.
- The `prefix` and `line` used by the RTS verifier when an exception occurs during parsing.

### CODE REVIEW

File: src/rollback_to_stable/rts_btree_walk.c [hunk 1]
```diff
@@ -68,7 +68,9 @@ __rts_btree_walk_page_skip(
                   WT_RTS_VERB_TAG_SKIP_DEL_NULL "ref=%p: deleted page walk skipped", (void *)ref);
             else {
                 __wt_verbose_multi(session, WT_VERB_RECOVERY_RTS(session),
-                  "%p: deleted page walk skipped page_del %s", (void *)ref,
+                  WT_RTS_VERB_TAG_STABLE_PG_WALK_SKIP
```
Comment (kommiharibabu) [inline src/rollback_to_stable/rts_btree_walk.c:71]: I believe we need a unique verbose tag to identify the message. Using an existing tag may not work.
I prefer the tag name to contain `DEL` as we are dealing with the deleted pages.

Comment (etienneptl) [reply src/rollback_to_stable/rts_btree_walk.c]: I created a new tag and used it, see d416ce9158859eb9bafab63d0a8d55fa5cee122f.

**Review by kommiharibabu — APPROVED** [review]
LGTM

File: tools/rts_verifier/operation.py [hunk 2]
```diff
@@ -96,7 +96,7 @@ def __extract_pointer(self, prefix, line):
             matches = re.search(f'{prefix}=(0x[A-Za-z0-9]+)', line)
 
         if matches is None:
-            raise Exception("failed to parse address string")
+            raise Exception(f"failed to parse address string with prefix '{prefix}' in '{line}'")
```
Comment (agorrod) [inline tools/rts_verifier/operation.py:99]: Could you help me understand this? I'm not familiar with the `f"<string>"` semantic.

Comment (etienneptl) [reply tools/rts_verifier/operation.py:99]: It is just a way to print variables with Python 3.x, see the documentation [here](https://docs.python.org/3/reference/lexical_analysis.html#f-strings). I believe we are not always consistent in the way we print variables in a string when using Python, unfortunately.

I added the variables to make the debugging experience easier.

**Review by clarissecheah — APPROVED** [review]
LGTM!

**Review by agorrod — APPROVED** [review]
Thanks @etienneptl lgtm.

**Comment by etienneptl** [comment]
evergreen merge

**Pull request merged**